packages/storage-plugin/src/symbols.ts

```diff
@@ -25,7 +25,7 @@ export interface NgxsStoragePluginOptions {
     /**
      * Version to key off.
      */
-    version: number | string;
```

Comment:
it's breaking changes

Reply:
Yes this is a breaking change, but as of now if you install the plugin and don't have a version in your default state, it will save the default state without a version and then the check doesn't work. I changed it so that it looks for a higher version in the migrations array, for which I need a number.

Reply:
I can change this PR to fix the bug (not finding a version in state prevents migrating) in another way and request a new feature if you'd prefer that.

Reply:
You can use parseInt or Number for convert version key to integer in your code)

Reply:
But that would imply that you can use a string, when the comparison is on numbers.